test/core/Matrix.js

```diff
@@ -107,10 +107,60 @@ describe('PIXI.Matrix', function ()
         expect(m1.ty).to.equal(m2.ty);
     });
 
+    it('should prepend matrix', function ()
+    {
+        const m1 = new PIXI.Matrix();
+        const m2 = new PIXI.Matrix();
+
+        m2.set(2, 3, 4, 5, 100, 100);
```

Comment:
I'd suggest setting `ty` to `100` or something not the same as `tx` to make the assert more useful for checking the parameters vs properties.